Src/FluentAssertions/Extensions/DictionaryExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace FluentAssertions.Extensions
```

Comment:
I would keep this in the root namespace so that it becomes discoverable.

Reply:
Moved to root namespace